recipes-bsp/alsa-state/files/asound-cubox-i.conf

```diff
@@ -0,0 +1,459 @@
+defaults.pcm.rate_converter "linear"
```

Comment:
The correct way to do this is by adding the file inside the <machine> directory.

So your file will be at `recipes-bsp/alsa-state/files/cubox-i/asound.conf` and your  `.bbappend` will add:

```
FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Your `do_install_append` can be dropped.